lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -298,7 +298,6 @@
     @include transition(all 0.15s linear 0s);
     @include clearfix();
     @extend %ui-depth2;
-    position: relative;
```

Comment:
This was causing the course title to appear above the dropdown. Removing this didn't appear to have negative (or any) visual consequences.